Dockerfile

```diff
@@ -1,15 +1,11 @@
-FROM openjdk:17-slim
-CMD ["/usr/local/openjdk-17/bin/java", "-jar", "/opt/ssdc-rm-response-operations.jar"]
+FROM eclipse-temurin:17-jdk-alpine
 
-RUN groupadd --gid 999 response-operations && \
-    useradd --create-home --system --uid 999 --gid response-operations response-operations
-
-RUN apt-get update && \
-apt-get -yq install curl && \
-apt-get -yq clean && \
-rm -rf /var/lib/apt/lists/*
```

Comment:
Losing curl on this image breaks the docker-dev health check, we need to keep it or do something else for the local health check.